last_seen.py

```diff
@@ -0,0 +1,27 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+class LastSeen(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.seen = self.open_storage('last_seen')
+
+    @event
+    def seen_evt(self, event):
+        events = ['quit', 'part', 'kick', 'join', 'rename', 'topic']
```

Comment:
@JakubNvk why is `message` not in here? What if the user is stil connected but not active in a channel? Wouldn't you want to see his or her last activity which might easily be sending a message?